lib/feed/widgets/feed_fab.dart

```diff
@@ -51,9 +51,6 @@ class FeedFAB extends StatelessWidget {
                 semanticLabel: singlePressAction.title,
                 size: 35,
               ),
-              onSlideDown: () {
```

Comment:
I removed this because this does not do anything. The slide down logic is located in the `GestureFab`.